test/types/coerce/diten/test_uint64_op_int_error2.prediff

```diff
@@ -0,0 +1 @@
+../../../../util/test/prediff-obscure-module-linenos
```

Comment:
Is this necessary, since the test doesn't seem to have any module line numbers?  Or does the test generate some and this script removes them entirely?

Reply:
It was unnecessary and only relevant with earlier designs. I've removed it.